mtags/src/main/scala-2/scala/meta/internal/metals/SemanticdbDefinition.scala

```diff
@@ -42,7 +43,7 @@ object SemanticdbDefinition {
   ): Unit = {
     input.toLanguage match {
       case Language.SCALA =>
-        val mtags = new ScalaToplevelMtags(input, true) {
+        val mtags = new ScalaToplevelMtags(input, true, dialects.Scala213) {
```

Comment:
Yep, but this method is used in search. 
I will create an issue because it would be better to do it in a separate PR. Getting the right dialect at call-sites of this method isn't trivial now.

Reply:
Sure, makes sense